README.md

```diff
@@ -123,6 +123,18 @@ my-dynamodb-journal.aws-client-config.max-connections = <your value here>
 
 Changing this number changes both the number of concurrent connections and the used thread-pool size.
 
+## Retry behavior
+
+This plugin uses exponential backoff when 50X errors from DynamoDB would occur. This includes network glitches and provisioned throughput exceptions (request rate is too high).
```

Comment:
@spangaer can you change this to mention that this Retry behavior was added in v1.1.0?

Reply:
I believe this to be addressed with a1fb159